appinventor/components-ios/src/LocationSensor.swift

```diff
@@ -215,6 +215,9 @@ open class LocationSensor: NonvisibleComponent, CLLocationManagerDelegate, Lifec
     geocoder.geocodeAddressString(address) { [self] placemarks, error in
       if let error = error {
         print("Error geocoding address: \(error.localizedDescription)")
+        // Note in theory we could have a single error here, but the Android version will produce an error for each dimension.
+        self._form?.dispatchErrorOccurredEvent(self, "LatitudeFromAddress", ErrorMessage.ERROR_LOCATION_SENSOR_LATITUDE_NOT_FOUND, address)
+        self._form?.dispatchErrorOccurredEvent(self, "LongitudeFromAddress", ErrorMessage.ERROR_LOCATION_SENSOR_LATITUDE_NOT_FOUND, address)
```

Comment:
Error messages for latitude and longitude are the same. The second string should be set to longitude.